app/models/application_record.rb

```diff
@@ -137,4 +147,29 @@ def self.ethnicity_eeo_data(records)
       [ethnicity, specific_records.count, specific_records.interview_count]
     end
   end
+
+  def self.notification_emails(records)
+    emails = Hash.new { |h, k| h[k] = {} }
+    records.each do |record|
+      emails = build_email_hash(emails, record)
+    end
+    send_emails(emails)
+  end
+
+  def self.send_emails(emails)
+    if emails.present?
+      emails.each_key do |email|
+        JobappsMailer.saved_applications_notification(emails[email], email)
+      end
+    end
+  end
+
+  def self.build_email_hash(emails, record)
```

Comment:
I think this method could use a comment explaining what it returns given its inputs.